app/assets/stylesheets/ontologies.scss

```diff
@@ -195,7 +195,7 @@ $widget-table-border-color: #EFEFEF;
 #bd_content .sidebar {
   overflow-x: auto;
   white-space: nowrap;
-  min-width: 35%;
+  min-width: 29%;
```

Comment:
why is this fixing the issue?

Reply:
since we don't have an explicite width, the element base on this and this is the value of the width for the other once (instances, ...)

Reply:
yes but you changed just 35% to 29%, if id did not work for `35%` why does it for `29%`? 
the real fix was not already this https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/649

Reply:
https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/649 this one is related to the right side, 
and this current PR is related to the width of the container of the tree view (we have different width when we switch from concepts to instances for example)

Reply:
>  if it did not work for 35% why does it for 29%?

Reply:
I don't really know how, but I tested it and it does work. So I merge.